docs/upgrading-solidus/v4.6.mdx

```diff
@@ -0,0 +1,99 @@
+- **Table Improvements**: Fixed sorting functionality and better semantic markup
+
+## 📦 Component Updates
+
+### Solidus Core
+- **Migration Improvements**: All migrations can now be properly rolled back
+- **Rails Compatibility**: Better support for multiple Rails versions with proper migration inheritance
+- **Database Relations**: Enhanced addressbook with proper foreign keys and associations
+- **Logging**: Replaced `puts` statements with proper Rails logging throughout tasks and generators
+
+### Solidus Admin
+- **UI Performance**: Optimized select component performance for large datasets
+- **Component Library**: New reusable UI components (Alert, improved Select)
+- **Form Handling**: Better textarea support and form validation
+- **Lookbook Integration**: Fixed installation issues for component documentation
+
+### Solidus Backend
+- **Payment Refunds**: Fixed critical issue with refunding uncompleted payments
+- **AJAX Performance**: Added intelligent 500ms delay for Select2 AJAX requests to reduce server load
+
+### Solidus API
+- **Helper Refactoring**: Improved user role and authentication helper methods
+- **Performance**: Better caching for current API user lookups
+
+### Solidus Promotions
+- **New Calculator**: Added PercentWithCap calculator for more flexible discount options
+- **Localization**: Improved calculator label translations using Rails i18n
+- **Flickwerk Integration**: Better patch management in initializers
+
+## 🔧 Technical Improvements
+
+### Testing & Development
+- **Browser Testing**: Migrated from Chrome to Firefox for more stable system specs
+- **Flaky Test Fixes**: Resolved numerous intermittent test failures
+- **Linting**: Improved code quality with updated linting rules
+
+### Dependencies
+
+- **Updated ImportMap Rails**: Upgraded to v2 for better asset management
+- **Migration Standards**: Enforced consistent migration versioning across the framework
+
+## 🐛 Bug Fixes
+
+- Fixed typos throughout the codebase ([#6207](https://github.com/solidusio/solidus/pull/6207))
+- Resolved currency display issues in admin order listings ([#5929](https://github.com/solidusio/solidus/pull/5929))
+- Fixed metadata migration compatibility with custom user classes ([#6157](https://github.com/solidusio/solidus/pull/6157))
+- Corrected form tag closure issues in table components ([#6172](https://github.com/solidusio/solidus/pull/6172))
+- Improved deprecation warning messages for better developer experience ([#6163](https://github.com/solidusio/solidus/pull/6163))
+- Fixed documentation comments to reflect default settings ([#6171](https://github.com/solidusio/solidus/pull/6171))
+
+## 🔄 Breaking Changes
+
+- Email sending behavior has changed from direct mailer calls to event subscribers. While backward compatible, custom email logic may need updates
+- Migration version enforcement may require attention if you have custom migrations with incorrect versions
```

Comment:
I don't think this is true. Migration versions should only be enforced for Solidus developers, not application developers.